spec/ttml1.xml

```diff
@@ -4716,15 +4716,43 @@ as a style property only to those element types indicated in the following table
 </tbody>
 </table>
 <p>If the value of this attribute is <code>normal</code>, then the
-computed value of the style property must be considered to be no less than the
-size of the largest font size that applies to the element and its
-descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>.</p>
+resolved computed value of this style property should be determined as follows:</p>
+<olist>
+<item><p>
+Let <emph>P</emph> be the <loc href="#content-vocabulary-p"><el>p</el></loc> element to which this style property applies.
+</p></item>
+<item><p>
+Let <emph>FF</emph> be the computed value of the <loc href="#style-attribute-fontFamily"><att>tts:fontFamily</att></loc>
+style property of <emph>P</emph>.
+</p></item>
+<item><p>
+Let <emph>FS</emph> be the computed value of the <loc href="#style-attribute-fontSize"><att>tts:fontSize</att></loc> style property of <emph>P</emph>.
+</p></item>
+<item><p>
+Let <emph>F0</emph> be the first font obtained when sequentially mapping each font family in <emph>FF</emph> to a set of available fonts,
+where this set of available fonts is constrained as needed to satisfy the computed values of the
+<loc href="#style-attribute-fontStyle"><att>tts:fontStyle</att></loc> and 
+<loc href="#style-attribute-fontWeight"><att>tts:fontWeight</att></loc>
+style properties of <emph>P</emph>.
+</p></item>
+<item><p>
+If <emph>F0</emph> is associated with font metrics that specify altitude <emph>A</emph>, descent <emph>D</emph>, and line gap <emph>G</emph>,
+then set <emph>LH</emph> to the sum of scaled(<emph>A</emph>), scaled(<emph>D</emph>), and scaled(<emph>G</emph>),
+where scaled(<emph>X</emph>) denotes font metric <emph>X</emph> scaled according to font size <emph>FS</emph>.
+</p></item>
+<item><p>Otherwise, <emph>LH</emph> is considered to be implementation dependent; however, in the absence of implementation specific requirements,
+<emph>LH</emph> is recommended to be set to 125% of <emph>FS</emph>.
+</p></item>
+<item><p>
+Set the resolved computed value of this style property to <emph>LH</emph>.
+</p></item>
+</olist>
 <note role="elaboration">
-<p>It is recommended that presentation processors use a computed line height that is
-125% of the size of the largest applicable font size when the value <code>normal</code>
-is specified. The choice of 125% allows use of a font size of <code>0.8c</code> in order
```

Comment:
I see no reason to vary from the TTML2 text, so I oppose restoring that note.

Reply:
I do not see the note in TTML2 re: `0.8c`. I also think it is misleading since there is simply no guarantee that `125%` will be used. Also, why is `0.8c` relevant, I forget.

Reply:
@nigelmegitt Ok. However, as discussed before, the 125% is only a suggestion, and setting the fontsize to 0.8c is no guarantee that the actual line height will be exactly 1c.